app/utils/file/index.ts

```diff
@@ -477,52 +477,52 @@ export const fileExists = async (path: string) => {
 };
 
 export const hasWriteStoragePermission = async (intl: IntlShape) => {
-    if (Platform.OS === 'ios') {
-        return true;
-    }
-
-    const storagePermission = PERMISSIONS.ANDROID.WRITE_EXTERNAL_STORAGE;
-    let permissionRequest;
-    const hasPermissionToStorage = await Permissions.check(storagePermission);
-    switch (hasPermissionToStorage) {
-        case Permissions.RESULTS.DENIED:
-            permissionRequest = await Permissions.request(storagePermission);
-            return permissionRequest === Permissions.RESULTS.GRANTED;
-        case Permissions.RESULTS.BLOCKED: {
-            const applicationName = DeviceInfo.getApplicationName();
-            const title = intl.formatMessage(
-                {
-                    id: 'mobile.storage_permission_denied_title',
+    if (Platform.OS === 'android' && Platform.Version < 33) {
```

Comment:
Is this the same function as `hasStoragePermission` with just changing the version, the permission itself, and the alert message?

I wonder if we should unify them.

Also... sanity check... storage check is on version 32, this one is on version 33. Is that correct?

Reply:
Is the same, although not needed for SDK 33 as that permission no longer exists